go/arrow/internal/arrdata/arrdata.go

```diff
@@ -997,6 +998,57 @@ func makeUnionRecords() []arrow.Record {
 		array.NewRecord(schema, []arrow.Array{sparse2, dense2}, -1)}
 }
 
+func makeRunEndEncodedRecords() []arrow.Record {
+	mem := memory.NewGoAllocator()
+	schema := arrow.NewSchema([]arrow.Field{
+		{Name: "ree16", Type: arrow.RunEndEncodedOf(arrow.PrimitiveTypes.Int16, arrow.BinaryTypes.String)},
+		{Name: "ree32", Type: arrow.RunEndEncodedOf(arrow.PrimitiveTypes.Int32, arrow.PrimitiveTypes.Int32)},
+		{Name: "ree64", Type: arrow.RunEndEncodedOf(arrow.PrimitiveTypes.Int64, arrow.BinaryTypes.Binary)},
+	}, nil)
+
+	schema.Field(1).Type.(*arrow.RunEndEncodedType).ValueNullable = false
+	mask := []bool{true, false, false, true, true}
```

Comment:
Is it a validity or a nullness mask? Might be worth calling it either `is_valid` or `is_null` to make things more readable.

Reply:
done